openassessment/xblock/static/js/src/lms/components/WaitingStepList.jsx

```diff
@@ -15,10 +34,53 @@ const WaitingStepList = ({ studentList, refreshData }) => {
     <Button onClick={() => refreshData()}>{gettext('Refresh')}</Button>
   );
 
+  const FindLearnerAction = (props) => {
+    const { tableInstance: { selectedFlatRows = [] } } = props || { tableInstance: {} };
+
+    const selectedFlatRowsLength = selectedFlatRows.length;
+
+    if (selectableLearners && selectedFlatRowsLength) {
+      const [row] = selectedFlatRows;
+      const {
+        values: { username },
+      } = row;
+
+      const invalidSelection = selectedFlatRowsLength !== 1;
+
+      const handleFindLearnerClick = () => {
+        if (findUsername) {
+          findUsername(username);
+        }
+      };
+
+      return (
+        <>
+          {invalidSelection && (
+            <Form.Control.Feedback type="invalid" className="my-2">
+              {gettext('You must select one row')}
+            </Form.Control.Feedback>
+          )}
+          <Button
+            iconAfter={Search}
+            variant="brand"
+            data-testid="find-learner-button"
+            disabled={invalidSelection}
+            onClick={handleFindLearnerClick}
+          >
+            {gettext('Search learner')}
```

Comment:
I tested the translation and it didn't work:
![image](https://github.com/openedx/edx-ora2/assets/64440265/5f7dd932-37db-4797-bfbc-17692715e5aa)
Maybe we're missing something. I'll research a bit but I'm not sure I'll find something since I'm not familiar with this repo. We can ask for help later when this PR is ready for review!